pkg/controller/volume/expand/expand_controller.go

```diff
@@ -447,3 +448,9 @@ func (expc *expandController) GetSubpather() subpath.Interface {
 	// not needed for expand controller
 	return nil
 }
+
+
+func (adc *expandController) GetFilteredDialOptions() *proxyutil.FilteredDialOptions {
+	// not needed for expand controller
```

Comment:
same thing if the plugins implement Expand

Reply:
Done